.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+yarn.lock   * @VKCOM/vk-sec
+.yarnrc     * @VKCOM/vk-sec
```

Кажется, * тут лишние.

```suggestion
yarn.lock   @VKCOM/vk-sec
.yarnrc     @VKCOM/vk-sec
```

---

Поправил, спасибо!